nextflow_schema.json

```
@@ -619,6 +624,27 @@
                 }
             }
         },
+        "virus_identification_options": {
+            "title": "Virus identification options",
+            "type": "object",
+            "default": "",
+            "properties": {
+                "run_genomad": {
+                    "type": "boolean",
+                    "description": "Identify viral sequences in assemblies"
```

Using what? Will there be other options in teh future?

---

No other options for virus identification planned at the moment. Only perhaps CheckV (which checks virus quality) which will be downstream.
```suggestion
                    "description": "Identify viral sequences in assemblies using geNomad"
```

---

Yes, fine for now :+!